doc/classes/Dictionary.xml

```diff
@@ -98,7 +98,7 @@
 		[codeblocks]
 		[gdscript]
 		# This is a valid dictionary.
-		# To access the string "Nested value" below, use `my_dir.sub_dir.sub_key` or `my_dir["sub_dir"]["sub_key"]`.
+		# To access the string "Nested value" below, use `my_dict.sub_dict.sub_key` or `my_dict["sub_dict"]["sub_key"]`.
```

Comment:
There is another typo in here, actually. It should be `sub_dict`. 🙃 

And same in C# snippet below.